dvc/remote/local.py

```diff
@@ -67,6 +67,13 @@ def cache_dir(self, value):
     def supported(cls, config):
         return True
 
+    @cached_property
+    def cache_path(self):
+        return os.path.abspath(self.cache_dir)
```

Comment:
If path is not abspath or relpath from current working dir, posix `lstat()` syscall runtime doubles. `Pathinfo.__str__` uses relpath from cwd, but since cwd may change during runtime (like in some of our erepo tests), we can't cache `relpath(self.cache_dir)` and we don't want to compute it each time, so we just use abspath.